example/manifest_branch.yaml

```diff
@@ -0,0 +1,50 @@
+# Version specifies which version is being being branched
+# The new branch will be of the form release-${version}
+version: 1.8
+
+# We need to set the docker in the manifest so it can be used as HUB in
+# the release builder standard environment. HUB is required to be set for
+# calling `make`.
+docker: docker.io/istio
```

Comment:
can this be called `hub` instead?

Reply:
It would need to be changed across all the commands as it's part of what release and publish use. I actually tried to remove it from the files, but the HUB environment variable gets set to this value and is used.  I'll see how big of an issue this is.

Reply:
I'll see how big this is in a later commit, and maybe it should be in a separate PR.